Tests/SwiftCurrent_SwiftUITests/GenericConstraintTests.swift

```diff
@@ -1158,26 +1112,27 @@ final class GenericConstraintTests: XCTestCase, View {
         }
         let expectedArgs = UUID().uuidString
 
-        let workflowView = WorkflowLauncher(isLaunched: .constant(true), startingArgs: expectedArgs) {
-            thenProceed(with: FR0.self) {
-                thenProceed(with: FR1.self) {
-                    thenProceed(with: FR2.self)
-                }
-            }
-        }
-        let expectViewLoaded = ViewHosting.loadView(workflowView).inspection.inspect { view in
-            XCTAssertNoThrow(try view.find(FR0.self).actualView().proceedInWorkflow())
-            try view.actualView().inspectWrapped { view in
-                XCTAssertNoThrow(try view.find(FR1.self).actualView().proceedInWorkflow())
-                try view.actualView().inspectWrapped { view in
-                    XCTAssertNoThrow(try view.find(FR2.self))
+        let workflowView = try await MainActor.run {
+            WorkflowLauncher(isLaunched: .constant(true), startingArgs: expectedArgs) {
+                thenProceed(with: FR0.self) {
+                    thenProceed(with: FR1.self) {
+                        thenProceed(with: FR2.self)
+                    }
                 }
             }
-        }
-        wait(for: [expectViewLoaded], timeout: TestConstant.timeout)
+        }.hostAndInspect(with: \.inspection).extractWorkflowItem()
+
```

Comment:
small gripe, can we remove these empty lines? Or is there a good case for them?

Reply:
Fixed